diff -r 8bff3332e34f Cython/Compiler/Code.py
--- a/Cython/Compiler/Code.py	Tue Feb 02 02:10:32 2010 -0800
+++ b/Cython/Compiler/Code.py	Thu Feb 04 19:33:59 2010 -0800
@@ -667,7 +667,7 @@
         decls_writer = self.parts['decls']
         for _, cname, c in c_consts:
             decls_writer.putln('static char %s[] = "%s";' % (
-                cname, c.escaped_value))
+                cname, StringEncoding.split_docstring(c.escaped_value)))
             if c.py_strings is not None:
                 for py_string in c.py_strings.itervalues():
                     py_strings.append((c.cname, len(py_string.cname), py_string))
